### PR TITLE
Updating the what's new page

### PR DESCRIPTION
**Pull request by sfshaza2**
As it says

### CODE REVIEW

**Comment by domesticmouse** [comment]
Looks like a formatting issue:

```
[11ty] Original error stack trace: invalid value expression: "% comment %", file:./src/content/release/whats-new.md, line:94, col:3
```

File: src/content/release/whats-new.md [hunk 1]
```diff
@@ -23,6 +23,87 @@ and review the [Dart changelog][].
...
+The following articles were published on the [Flutter Medium][]
+publication since Flutter 3.19:
+
+{{% comment %}}
+LATER:
+- Predictive back support (Justin) (still experimental)
+list new codelabs
+
+also update codelabs page
+{{% endcomment %}}
```
Comment (parlough) [inline src/content/release/whats-new.md:105]:
```suggestion
{% comment %}
LATER:
- Predictive back support (Justin) (still experimental)
list new codelabs

also update codelabs page
{% endcomment %}
```

**Review by parlough — APPROVED** [review]

**Comment by sfshaza2** [comment]
THANKS, @parlough!!!

**Comment by sfshaza2** [comment]
@parlough, it still seems to be broken...

File: src/content/release/whats-new.md [hunk 2]
```diff
@@ -23,6 +23,87 @@ and review the [Dart changelog][].
...
+
+[Adaptive and Responsive design]: /ui/adaptive-responsive
+[Casual Games Toolkit]: /ui/resources/games-toolkit
+[Conditionally bundling assets based on flavor]: /deployment/flavors#conditionally-bundling-assets-based-on-flavor
+[First week experience of Flutter]: /get-started/fwe
+[Flutter install]: /get-started/install
+[Flutter web app initialization]: /platform-integration/web/bootstrapping
+[website README]: {{site.github}}/flutter/website/?tab=readme-ov-file#flutter-documentation-website
+[Support for WebAssembly (Wasm)]: /platform-integration/web/wasm
+[Transforming assets at build time]: /ui/asset/asset-transformer page
```
Comment (parlough) [inline src/content/release/whats-new.md:91]:
```suggestion
[Transforming assets at build time]: /ui/assets/asset-transformation
```

File: src/content/release/whats-new.md [hunk 3]
```diff
@@ -23,6 +23,87 @@ and review the [Dart changelog][].
...
+[Validate deep links]: /tools/devtools/deep-links
+
+**Articles**
+
+The following articles were published on the [Flutter Medium][]
```
Comment (parlough) [inline src/content/release/whats-new.md:96]: It looks like "Flutter Medium" might be missing a link definition.

**Comment by sfshaza2** [comment]
Thanks, @domesticmouse! I've remove it and am trying again.

File: src/content/release/whats-new.md (hunk 3 above)
Comment (sfshaza2) [reply src/content/release/whats-new.md:96]: Yeah, I already fixed that.  :)

**Pull request merged**